src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -847,7 +847,9 @@ void SignalDetectorClass::processMessage()
 	
 	if (!m_truncated)  // Todo: Eventuell auf vollen Puffer prüfen
 	{
+		bool _success=success;  // save success because it is set to false in reset()
 		reset();
+		success=_success;  // restore success
```

Comment:
Do you have any more information, why you think this ist needed?
Maybe some data that isn't processed without?

Reply:
The PIN_LED is turned in loop() with the following code:
```
state = musterDec.decode(&aktVal);
if (state) blinkLED = true; //LED blinken, wenn Meldung dekodiert
```
 and the bool success (defined in signalDecoder.cpp )  is the return of `musterDec.decode`